packages/data-context/src/sources/HtmlDataSource.ts

```diff
@@ -35,7 +35,7 @@ export class HtmlDataSource {
         )
       } catch (e) {
         err = e
-        await this.ctx.util.delayMs(1000)
+        await new Promise((resolve) => setTimeout(resolve, 1000))
```

Comment:
might be worth just using `Bluebird.timeout` here and elsewhere in this PR... so when we strip `Bluebird` out these timeouts are more easily discovered and replaced with a shared helper. Blue and I recently discussed this here: https://github.com/cypress-io/cypress/pull/21328#discussion_r865463268

Reply:
I don't think we use Bluebird at all in the `data-context` package currently

Reply:
:man_shrugging: